Core/include/Acts/Propagator/detail/GenericDenseEnvironmentExtension.hpp

```diff
@@ -337,42 +355,34 @@ struct GenericDenseEnvironmentExtension {
     // The following comment lines refer to the application of the time being
     // treated as a position. Since t and qop are treated independently for now,
     // this just serves as entry point for building their relation
-    //~ double dtpp1dl = -state.options.mass * state.options.mass * qop[0] *
-    //~ qop[0] *
-    //~ (3. * g + qop[0] * dgdqop(energy[0], state.options.mass,
-    //~ state.options.absPdgCode,
+    //~ double dtpp1dl = -mass * mass * qop[0] * qop[0] *
+    //~ (3. * g + qop[0] * dgdqop(energy[0], .mass,
+    //~ absPdg,
     //~ state.options.meanEnergyLoss));
 
-    double dtp1dl = qop[0] * state.options.mass * state.options.mass /
-                    std::hypot(1, qop[0] * state.options.mass);
+    double dtp1dl = qop[0] * mass * mass / std::hypot(1, qop[0] * mass);
```

Comment:
Why is `std::` fine here?

Reply:
not sure but I will try to make it consistent